internal/parsing/parsing.go

```diff
@@ -1,9 +1,32 @@
 package parsing
 
+import (
+	"encoding/json"
+	"fmt"
+
+	"github.com/RHEnVision/provisioning-backend/internal/clients/sources"
+)
+
 func IsHTTPStatus2xx(status int) bool {
 	return status >= 200 && status <= 299
 }
 
 func IsHTTPNotFound(status int) bool {
 	return status == 404
 }
+
+func ParseErrorNotFoundToJSON(errors sources.ErrorNotFound) ([]byte, error) {
+	j, err := json.Marshal(errors)
+	if err != nil {
+		return nil, fmt.Errorf("cant marshel array to json")
+	}
+	return j, nil
+}
+
+func ParseErrorBadRequestToJSON(errors sources.ErrorBadRequest) ([]byte, error) {
+	j, err := json.Marshal(errors)
+	if err != nil {
+		return nil, fmt.Errorf("cant marshel array to json")
+	}
+	return j, nil
+}
```

Comment:
Since these errors are specific to sources, could you put them there? This package looks like generic parsing.